java/spacewalk-java.changes.rjpmestre.new_maintanance_update_notification

```diff
@@ -0,0 +1 @@
+ - Fix SUMA/Uyuni update checker
```

Comment:
I would describe this differently:

```suggestion
 - Adapt SUMA/Uyuni new version update checker for containers
```

With some imagination this may hint changelog reader that this is not just arbitrary fix.